src/toggle/Toggle.tsx

```diff
@@ -0,0 +1,74 @@
+import "./_toggle.scss";
+
+import React, {useCallback, useState} from "react";
+import classNames from "classnames";
+
+import ToggleItem from "./item/ToggleItem";
+import {ToggleContext} from "./util/ToggleContext";
+
+export interface ToggleProps {
+  children: React.ReactNode;
+  onToggle?: (dataId: string) => void;
+  selectedItems?: string[];
+  isMultiple?: boolean;
+  position?: "vertical" | "horizontal";
+  isDisabled?: boolean;
+  customClassName?: string;
+}
+
+function Toggle({
+  children,
+  onToggle,
+  isMultiple = false,
+  selectedItems = [],
+  position = "horizontal",
+  isDisabled,
+  customClassName
+}: ToggleProps) {
+  const toggleClassName = classNames("toggle", customClassName, {
+    "toggle--is-horizontal": position === "horizontal",
+    "toggle--is-vertical": position === "vertical",
+    "toggle--is-disabled": isDisabled
+  });
+  const [selectedToggleItemsState, setSelectedToggleItemsState] = useState<string[]>(
+    selectedItems
```

Comment:
this should be `initialSelectedItems`. IF they provide `selectedItems`, then we should pass `selectedItems` directly to the context instead of using `selectedToggleItemsState`. Please add an example to the Storybook that shows the use of `selectedItems` – you will see that your current implementation is not working properly in that case.